JSON_Bourne/get_webpage.py

```diff
@@ -0,0 +1,112 @@
+from lxml import html
+import requests
+import json
+
+
+def get_info(url):
```

Comment:
This method tries to build objects that are basically tuples of (title,status,value,alarm). Doing this by looping over the index of several disconnected arrays seems like quite a non-Pythonic way to do things. Seems more like Fortran. It would be better to create an info class with the above 4 properties and then populate instances using the information from the arrays.